release.md

```diff
@@ -0,0 +1,14 @@
+## Create composeml release on github
```

Comment:
what about instructions for releasing to pypi?

Reply:
Updated instructions with release process in [this commit.](https://github.com/FeatureLabs/compose-ml/pull/23/commits/52480748697ea8619c5627f6fd273634211ca52c#diff-6cd007d62737e5a12b005b4414d7b899R17)